Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -2559,6 +2559,7 @@ PhysicalParticleContainer::PushP (int lev, Real dt,
     if (do_not_push) { return; }
 
     const std::array<amrex::Real,3>& dx = WarpX::CellSize(std::max(lev,0));
+    const amrex::XDim3 dinv = amrex::XDim3{1._rt/dx[0], 1._rt/dx[1], 1._rt/dx[2]};
```

Comment:
These lines are repeated quite a lot. Would it make sense to directly grab `dinv` from `WarpX` instead of every time needing both lines?
```
    const std::array<amrex::Real,3>& dx = WarpX::CellSize(std::max(lev,0));
    const amrex::XDim3 dinv = amrex::XDim3{1._rt/dx[0], 1._rt/dx[1], 1._rt/dx[2]};
```

Reply:
Yes, definitely. I could add a routine `InvCellSize` that returns an `XDim3`.

Reply:
Thanks!